### PR TITLE
Added live stream examples

### PR DESCRIPTION
**Pull request by gfronza**

### CODE REVIEW

File: src/test/java/com/bitmovin/api/examples/CreateAndStartLiveStreamWithVP9.java [hunk 1]
```diff
@@ -0,0 +1,352 @@
...
+		videoInputStream.setSelectionMode(StreamSelectionMode.AUTO);
+		videoInputStream.setPosition(0);
+
+		InputStream audioInputStream = new InputStream();
+		audioInputStream.setInputPath("live");
+		audioInputStream.setInputId(input.getId());
+		audioInputStream.setSelectionMode(StreamSelectionMode.AUTO);
+		audioInputStream.setPosition(1);
+
+		for (VideoProfile videoProfile : VIDEO_ENCODING_PROFILES) {
```
Comment (rkersche) [inline src/test/java/com/bitmovin/api/examples/CreateAndStartLiveStreamWithVP9.java]: `{` goes in next line

**Review by Tigraine — APPROVED** [review]
LGTM

**Review by rkersche — CHANGES_REQUESTED** [review]
Nice, please just move all `{` into the next line

**Comment by gfronza** [comment]
@rkersche Fixed the `{` position. Already changed my editor's code style :)

File: src/test/java/com/bitmovin/api/examples/CreateAndStartLiveStreamWithVP9.java [hunk 2]
```diff
@@ -0,0 +1,385 @@
...
+	private VP9VideoConfiguration createVP9VideoConfiguration(VideoProfile videoProfile)
+			throws BitmovinApiException, UnirestException, IOException, URISyntaxException
+	{
+		VP9VideoConfiguration videoConfig = new VP9VideoConfiguration();
+		videoConfig.setName(String.format("StreamDemoVP9%dp", videoProfile.height));
+		videoConfig.setBitrate(videoProfile.bitrate * 1000);
+		videoConfig.setRate(videoProfile.fps);
+		videoConfig.setWidth(videoProfile.width);
+		videoConfig.setHeight(videoProfile.height);
+		videoConfig.setId(String.format("vp9_%dp", videoProfile.height));
```
Comment (rkersche) [inline src/test/java/com/bitmovin/api/examples/CreateAndStartLiveStreamWithVP9.java]: IDs are read-only

**Review by rkersche — CHANGES_REQUESTED** [review]

**Comment by gfronza** [comment]
@rkersche I removed the `setId` call. Should be fine now.

**Review by rkersche — APPROVED** [review]

**Pull request merged**